### ISSUE TITLE
JDBC data store is always updated on terraform apply

### ISSUE THREAD

**Issue by awalmsley** [iwarapter/terraform-provider-pingfederate#51]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.14.6

### Affected Resource(s)

pingfederate_jdbc_data_store

### Terraform Configuration Files

```hcl
resource "pingfederate_jdbc_data_store" "this" {
  name           = "test"
  driver_class   = var.jdbc_driver_class
  user_name      = var.jdbc_username
  password       = var.jdbc_password
  max_pool_size  = 10
  connection_url = var.jdbc_connection_url
  connection_url_tags {
    connection_url = var.jdbc_connection_url
    default_source = true
  }
}
```

### Expected Behavior

Expected no update if Terraform configuration file not changed.

### Actual Behavior

```
$ terraform apply
pingfederate_jdbc_data_store.this: Refreshing state... [id=JDBC-E463B3A46AFCDB3261A3355C02D346A22A8F8642]

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # pingfederate_jdbc_data_store.this will be updated in-place
  ~ resource "pingfederate_jdbc_data_store" "this" {
      - encrypted_password           = "eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4Q0JDLUhTMjU2Iiwia2lkIjoiMEh4WHlYR2RvWSIsInZlcnNpb24iOiIxMC4xLjIuMCJ9..8TWYRLLJt_ZNigxlQsSyww.rBO00UfGWaA5ek1a0UeMyxfb3whxlsN44xYvAjCdFBg.AJE30fRpPDRiNYbqDAgY9A" -> null
        id                           = "JDBC-E463B3A46AFCDB3261A3355C02D346A22A8F8642"
        name                         = "test"
        # (10 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

pingfederate_jdbc_data_store.this: Modifying... [id=JDBC-E463B3A46AFCDB3261A3355C02D346A22A8F8642]
pingfederate_jdbc_data_store.this: Modifications complete after 0s [id=JDBC-E463B3A46AFCDB3261A3355C02D346A22A8F8642]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
$ terraform apply
pingfederate_jdbc_data_store.this: Refreshing state... [id=JDBC-E463B3A46AFCDB3261A3355C02D346A22A8F8642]

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # pingfederate_jdbc_data_store.this will be updated in-place
  ~ resource "pingfederate_jdbc_data_store" "this" {
      - encrypted_password           = "eyJhbGciOiJkaXIiLCJlbmMiOiJBMTI4Q0JDLUhTMjU2Iiwia2lkIjoiMEh4WHlYR2RvWSIsInZlcnNpb24iOiIxMC4xLjIuMCJ9..i0UVOvUPWrmyv-TuvJ22jw.elM-K-InzRF4FrPAnOnm12gaor53tfQsBl9Clmutz-E.sS5pt0bpiRkcCxZEtieDjQ" -> null
        id                           = "JDBC-E463B3A46AFCDB3261A3355C02D346A22A8F8642"
        name                         = "test"
        # (10 unchanged attributes hidden)

        # (1 unchanged block hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

pingfederate_jdbc_data_store.this: Modifying... [id=JDBC-E463B3A46AFCDB3261A3355C02D346A22A8F8642]
pingfederate_jdbc_data_store.this: Modifications complete after 0s [id=JDBC-E463B3A46AFCDB3261A3355C02D346A22A8F8642]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.
```

### Steps to Reproduce

repeat terraform apply without modifying Terraform configuration file

### References

* #0000

**Comment by iwarapter** [comment]
This is already fixed on master and will be in the next release.

**Comment by iwarapter** [comment]
Fixed in v0.0.11
